src/main/java/cn/nukkit/Player.java

```diff
@@ -4276,6 +4325,21 @@ public void kill() {
             this.dataPacket(pk);
         }
     }
+    
+    @PowerNukkitOnly
+    @Since("1.4.0.0-PN")
+    protected boolean isValidRespawnBlock(Block block) {
+        if (block.getId() == BlockID.RESPAWN_ANCHOR && block.getLevel().getDimension() == Level.DIMENSION_NETHER) {
```

Comment:
In that if, I think you forgot to add The End as valid for the Respawn Anchor.

Reply:
This assumption is not correct. Accourding to the [wiki](https://minecraft.gamepedia.com/Respawn_Anchor#Exploding) : 
`If the player attempts to set their spawn at a charged respawn anchor in the Overworld, the End, or customized dimensions in which they are disabled, the block explodes (and is destroyed) similarly to when a bed is used in the Nether or the End. The explosion has a power of 5, and sets fire to surrounding blocks.`